continuum/generators/scenarios_generator.py

```diff
@@ -0,0 +1,63 @@
+import abc
+from continuum.scenarios import _BaseScenario, create_subscenario
+import torch
```

Comment:
Can you sort a bit the import please?

BUILTIN PACKAGES
empty line
PYPI PACKAGES
empty line
INTERNAL IMPORTS

Reply:
only abc is builtin package right?

Reply:
yep